deepdoc/parser/mistral_parser.py

```
@@ -0,0 +1,487 @@
+            self._raise_for_status(r, "/files upload")
+            file_id = r.json().get("id")
+            r = requests.get(f"{self.base_url}/files/{file_id}/url", headers=self._headers(), params={"expiry": 24}, timeout=self.timeout)
+            self._raise_for_status(r, "signed-url fetch")
+            signed = r.json().get("url")
+            document = {"type": "document_url", "document_url": signed}
+            return self._post_ocr(self._ocr_payload(document, pages))
+        finally:
+            if file_id:
+                try:
+                    requests.delete(f"{self.base_url}/files/{file_id}", headers=self._headers(), timeout=self.timeout)
+                except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)